src/components/molecules/ChatPoll/ChatPoll.scss

```diff
@@ -11,6 +11,15 @@ button.ChatPoll__question {
   width: 100%;
   background-color: $secondary--light;
 
+  &:focus {
+    outline: none;
+  }
```

Comment:
Why are you doing this? Setting outline to `none` tends to create accessibility issues.

Reply:
fair, removed in 3e4c5f3310eeb05fddf062dec5155638f3125926